scripts/py_matter_idl/matter/idl/tests/inputs/large_all_clusters_app.matter

```diff
@@ -4104,7 +4104,7 @@ endpoint 0 {
     callback attribute overrunCount;
     callback attribute carrierDetect;
     callback attribute timeSinceReset;
-    ram      attribute featureMap default = 3;
+    ram      attribute featureMap default = 0;
```

Comment:
Likewise: why is this being changed?

Reply:
https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_idl/matter/idl/tests/inputs/large_all_clusters_app.matter#L1434

Ethernet network diagnostics has no features in this matter file. This was probably a copy & paste in zap somehow.